common/scripted_effects/00_scripted_effects.txt

```diff
@@ -18792,11 +18779,9 @@ china_war_spawn_troops_effect = {
 					home = event_target:spawn_province
 					troops = {
 						archers = { 136 136 }
-						light_infantry = { 134 134 }
-						heavy_infantry = { 102 102 }
-						pikemen = { 41 41 }
```

Comment:
I would summarize heavy and light infantry... or even give mages or something else.